examples/notebooks/petsc_eigenvalues_positive_Helmholtz.ipynb

```diff
@@ -0,0 +1,285 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1f0f103d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from petsc4py import PETSc\n",
+    "from psydac.linalg.utilities import petsc_to_psydac\n",
+    "\n",
+    "Ap = A.topetsc()\n",
+    "rhsp = rhs.topetsc()\n",
+    "\n",
+    "# initial solution\n",
+    "u = rhsp.copy()\n",
+    "u.zeroEntries() # set all entries to zero\n",
+    "\n",
+    "iterative_solver = PETSc.KSP().create(comm=comm) # create Krylov solver object\n",
+    "iterative_solver.setType(PETSc.KSP.Type.CG)\n",
+    "iterative_solver.setOperators(A=Ap, P=Ap) # set operator and preconditioner\n",
+    "preconditioner = iterative_solver.getPC()\n",
+    "preconditioner.setType(PETSc.PC.Type.JACOBI) # set Jacobi preconditioner, i.e. diagonal of matrix Ap\n",
+    "preconditioner.setUp()\n",
+    "\n",
+    "iterative_solver.setNormType(PETSc.KSP.NormType.UNPRECONDITIONED) #use unpreconditioned norm for convergence\n",
+    "\n",
+    "iterative_solver.setComputeEigenvalues(True) # compute eigenvalues during Krylov solver\n",
+    "def callback(ksp, iter, rnorm):\n",
+    "    if ksp.getComm().Get_rank() == 0:\n",
+    "        print(f'Iter {iter} | res = {\"{:.2e}\".format(rnorm)}')\n",
+    "\n",
+    "iterative_solver.setMonitor(callback) # set a callback function to print iteration and residual.\n",
+    "iterative_solver.setTolerances(atol=1e-5, rtol=1e-2, max_it=100)\n",
+    "iterative_solver.setInitialGuessNonzero(True) #if True it uses the solution vector as initial guess, otherwise it takes zero.\n",
+    "\n",
+    "iterative_solver.solve(rhsp, u)\n",
+    "\n",
+    "eigenvalues_approx = iterative_solver.computeEigenvalues() # the array size is the number of CG iterations\n",
+    "\n",
+    "# convert the solution back to PSYDAC\n",
+    "u_psydac = petsc_to_psydac(u, Wh.coeff_space)\n",
+    "uh = FemField(Wh, coeffs=u_psydac)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab581ba7",
+   "metadata": {},
+   "source": [
+    "## Step 5 : Plot approximative eigenvalues\n",
+    "The eigenvalues computed with the Krylov method are only approximative. The number of computed \"eigenvalues\" is the number of iterations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1a04a463",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "if comm is None or comm.Get_rank() == 0:\n",
+    "    fig,ax = plt.subplots(1,1)\n",
+    "    ax.set_title(f\"Eigenvalues estimated with CG {iterative_solver.getIterationNumber()} iterations\")\n",
+    "    im = ax.scatter(np.arange(eigenvalues_approx.size), eigenvalues_approx)\n",
+    "    plt.xlabel('index')\n",
+    "    fig.show()\n",
```

Comment:
It seems to me that calling `show()` in a notebook is not needed, and actually in the compiled documentation the plot appears twice. You can check this by opening the artifact as described in the PR's description